python/matcher_test.go

```diff
@@ -6,6 +6,7 @@ import (
 
 	"github.com/quay/zlog"
 
+	"github.com/google/go-cmp/cmp"
```

Comment:
Fix grouping -- once rebased on latest `main`, `git rebase -x etc/importfix.sh origin/main` should do it